docs/REST API/rest-api.md

```diff
@@ -487,17 +487,17 @@ for pagination. For a comprehensive list of activity types and detailed informat
     {
       "created_at": "2023-07-27T14:35:08Z",
       "id": 25,
-      "actor_full_name": "Anna Chao",
-      "actor_id": 3,
+      "actor_full_name": "Fleet",
```

Comment:
@rachaelshaw Could you take a look at this? For activities that user does not initiate (install software by policy automation, run script by policy automation, etc.), we plan to use `Fleet` as `actor_full_name`, and additionally add `fleet_initiated` flag.

The reason I'm asking is that @iansltx wants to implement that for global activities this sprint (the part with `Fleet` as the actor name), but we aim to use this pattern in the future. Let me know wdyt?